src/mode.lisp

```diff
@@ -104,6 +104,10 @@
 (defun current-global-mode ()
   *current-global-mode*)
 
+(defun current-global-mode-keyword-name ()
+  (alexandria:make-keyword
+   (mode-name (current-global-mode))))
```

Comment:
Why do you need to use current-global-mode-keyword-name instead of current-global-mode?

Reply:
https://github.com/lem-project/lem/pull/1176/files#r1444574079